src/routes/login.jsx

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+import { Box, Button, StyledOcticon, Text } from "@primer/react";
+import { MarkGithubIcon, TableIcon } from "@primer/octicons-react";
+
+// @ts-expect-error - look into how to define env vars in Vite
+const backendBaseUrl = import.meta.env.VITE_BACKEND_BASE_URL || "";
+
+export default function Login() {
+  return (
+    <Box
+      mt="4"
+      display="flex"
+      flexDirection="column"
+      alignItems="center"
+      justifyContent="center"
+      height="100vh"
+    >
+      <Box alignSelf="center" display="grid">
+        <StyledOcticon
+          icon={TableIcon}
+          size={52}
+          sx={{ ml: "auto", mr: "auto", mb: 3 }}
+        />
+
+        <Text mb="4" textAlign="center">
+          You need to sign in in order
+          <br />
+          to submit an issue.
+        </Text>
+
+        <Button
+          size="large"
+          leadingIcon={MarkGithubIcon}
+          as="a"
+          href={`${backendBaseUrl}/api/github/oauth/login?redirectUrl=${`http://localhost:8888`}`}
```

Comment:
⚠️ TODO: Before, we relied on the current `window.location`, but now it needs to point to the homepage (the route in charge of redirecting).

Reply:
Maybe we can introduce a new variable `import.meta.env.VITE_FRONTEND_BASE_URL` to get the redirect URL